src/twitter.ts

```diff
@@ -21,9 +21,11 @@ const getOpenseaCollection = async (
   const {
     party: { nftContractAddress, nftTokenId },
   } = event;
+  // collection buys do not have an associated nftTokenId. in this case, we default to tokenId 1
+  const tokenId = nftTokenId ? nftTokenId : 1;
   try {
     const r = await axios.get(
-      `https://api.opensea.io/api/v1/asset/${nftContractAddress}/${nftTokenId}`,
+      `https://api.opensea.io/api/v1/asset/${nftContractAddress}/${tokenId}`,
```

Comment:
i feel like at this point we should ditch the specific asset id lookup and just fetch the collection? feels a bit weird to rely on id 1

Reply:
the issue with this is that we need to know the opensea slug to get that information. let me check the gem api to see if we can get that from their endpoint